ui/raidboss/data/05-shb/raid/e5s.js

```diff
@@ -7,6 +7,206 @@
   },
   timelineFile: 'e5s.txt',
   triggers: [
+    {
+      id: 'E5S Stratospear Summons',
+      regex: Regexes.ability({ id: '4BA5', source: 'Ramuh', capture: false }),
+      regexDe: Regexes.ability({ id: '4BA5', source: 'Ramuh', capture: false }),
+      regexFr: Regexes.ability({ id: '4BA5', source: 'Ramuh', capture: false }),
+      regexJa: Regexes.ability({ id: '4BA5', source: 'ラムウ', capture: false }),
+      regexKo: Regexes.ability({ id: '4BA5', source: '라무', capture: false }),
+      delaySeconds: 5,
+      preRun: function(data) {
+        data.firstSpear = data.firstSpear || 1;
+      },
+      infoText: function(data) {
+        if (data.firstSpear == 1) {
+          return {
+            en: 'Look for small spear',
+            ko: '작은 지팡이 확인',
+          };
+        }
+      },
+      run: function(data) {
+        data.firstSpear++;
+      }
+    },
+    {
+      id: 'E5S Tribunal Summons',
+      regex: Regexes.startsUsing({ id: '4BAC', source: 'Ramuh', capture: false }),
+      regexDe: Regexes.startsUsing({ id: '4BAC', source: 'Ramuh', capture: false }),
+      regexFr: Regexes.startsUsing({ id: '4BAC', source: 'Ramuh', capture: false }),
+      regexJa: Regexes.startsUsing({ id: '4BAC', source: 'ラムウ', capture: false }),
+      regexKo: Regexes.startsUsing({ id: '4BAC', source: '라무', capture: false }),
+      preRun: function(data) {
+        data.adds = data.adds || 0;
+      },
+      infoText: function(data) {
+        if (data.adds != 0) {
+          return {
+            en: 'Look for adds',
+            ko: '쫄 위치 확인',
+          };
+        }
+      },
+      run: function(data) {
+        data.adds++;
+      }
+    },
+    {
+      id: 'E5S Fury\'s Bolt',
+      regex: Regexes.startsUsing({ id: '4BAA', source: 'Ramuh', capture: false }),
+      regexDe: Regexes.startsUsing({ id: '4BAA', source: 'Ramuh', capture: false }),
+      regexFr: Regexes.startsUsing({ id: '4BAA', source: 'Ramuh', capture: false }),
+      regexJa: Regexes.startsUsing({ id: '4BAA', source: 'ラムウ', capture: false }),
+      regexKo: Regexes.startsUsing({ id: '4BAA', source: '라무', capture: false }),
+      infoText: {
+        en: 'Fury\'s Bolt',
```

Comment:
Maybe this should say "pick up orb"?

Reply:
I wrote "Ramuh strengthened" in Korean, since one thing there donut AoE does not need orbs. So... I was wondering if which is better.

Reply:
Ah that's true! This is good for now.  Maybe in the future there can be some smarter "pick up orb" trigger too.